src/main/scala/beam/calibration/utils/DeleteSuggestion.scala

```diff
@@ -23,9 +23,8 @@ object DeleteSuggestion extends LazyLogging {
     BeamSigoptTuner.fetchExperiment(experimentId) match {
       case Some(_experiment) =>
         _experiment.suggestions().delete(suggestionId).call()
-      case None => {
+      case None =>
```

Comment:
This one is debatable - in fact it begs the question as to whether we should require brackets for anything that is not on a single line? I've seen subtle bugs where somebody adds something after the fact and their mental bracket expectations do not match the reality.